src/sdk/resources/policyLocationActions.ts

```diff
@@ -0,0 +1,42 @@
+import { IHttpClient } from '../http';
+import { getTypedResponse, BaseResource}  from './resource';
+import { ConfigurationPrompt, PolicyLocationAction, TrackingGroupLocationActions } from '../models';
+import { CancellationToken } from '../cancellation-token';
+
+export class PolicyLocationActionsResource extends BaseResource {
+	constructor(httpClient: IHttpClient) {
+		super(httpClient);
+	}
+
+	private getResourcePath(policyId: string, locationId: string): string {
+		return `policies/${policyId}/locations/${locationId}/actions`;
+	}
+
+	private getResourcePathSingle(policyId: string, locationId: string, trackingGroupActionId: string): string {
+		return `${this.getResourcePath(policyId, locationId)}/${trackingGroupActionId}`;
+	}
+
+	async getActions(policyId: string, locationId: string, params?: any, token?: CancellationToken): Promise<TrackingGroupLocationActions> {
+		const actions = await this.httpClient.get(this.getResourcePath(policyId, locationId), params, token);
+		return getTypedResponse<TrackingGroupLocationActions>(actions);
+	}
+
+	async getConfigurationPrompt(policyId: string, locationId: string, trackingGroupActionId: string, params?: any, token?: CancellationToken): Promise<ConfigurationPrompt> {
+		const prompt = await this.httpClient.get(this.getResourcePathSingle(policyId, locationId, trackingGroupActionId), params, token);
+		return getTypedResponse<ConfigurationPrompt>(prompt);
+	}
```

Comment:
```suggestion
	getConfigurationPrompt(policyId: string, locationId: string, trackingGroupActionId: string, params?: any, token?: CancellationToken): Promise<{ config_attributes: PromptAttributes }> {
		return this.httpClient.get(this.getResourcePathSingle(policyId, locationId, trackingGroupActionId), params, token);
	}
```

Reply:
Per this change: https://github.com/portalarchitects/skysync-vnext/pull/3606#discussion_r740387733